docs/sources/panels-visualizations/configure-tooltips/index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+keywords:
+  - grafana
+  - tooltips
+  - documentation
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure tooltips
+title: Configure tooltips
+description: Configure tooltips for your visualizations
+weight: 75
+---
+
+# Configure tooltips
+
+When you hover your cursor over a visualization, Grafana can display tooltips that contain more information about a data point like the exact time of a result. You can customize tooltips to to control how many series they include and the order of those values. You can also copy the content from tooltips to use elsewhere. Learn more about configuring tooltips in [Tooltip options](#tooltip-options).
+
+## Supported visualizations
+
+You can configure tooltips for the following visualizations:
+
+|                                  |                                  |
+| -------------------------------- | -------------------------------- |
+| [Bar chart][bar chart]           | [Status history][status history] |
+| [Heatmap][heatmap]               | [Time series][time series]       |
+| [Pie chart][pie chart]           | [Trend][trend]                   |
+| [State timeline][state timeline] |
```

Comment:
@adela-almasan `candlestick` as well? Or do we not count that since it's basically timeseries?

Reply:
Nevermind, it has one, just not configurable. Ignore me. 🫠